iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -99,6 +99,33 @@ static int64_t getVectorSize(FuncOp entryPointFn, ShapedType shapedType) {
   return getVectorSize(entryPointFn, byteWidth);
 }
 
+/// Returns minimum tiling sizes for each dimension. One dimension is possible
+/// to access at different element types. It determines the tiling sizes by
+/// looking into all the operands.
+static SmallVector<int64_t> getMinTilingSizesForEachDim(FuncOp entryPointFn,
+                                                        linalg::LinalgOp op) {
+  unsigned numLoops = op.getNumLoops();
+  SmallVector<int64_t> minTileSizes(numLoops, 1);
+  auto inputOutputOpOperands = op.getInputAndOutputOperands();
+  for (auto map : llvm::enumerate(op.getIndexingMaps())) {
```

Comment:
Is it possible to have operands within the LinalgOp body that are not going through the LinalgOp input/output operands (I don't remember this detail right now). Let's say a constant or something like that. IOW, I'm wondering if just looking at inputs/outputs is just enough or we should look at all the ops within the body.

Reply:
It is possible. IMO, it's fine at this moment. In this case we usually don't trucnate/extend the bit width. One of the operands (which does computation with constants) is considered.